File: duplicate.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        l = 1
        for i in range(1,len(nums)):
            if nums[i]!= nums[i-1]:
                nums[l] = nums[i]
                l += 1
        
s = Solution()
nums = [0,0,1,1,1,2,2,3,3,4]
s.removeDuplicates(nums)
print(nums)

File: duplicate.cpp
#include <vector>
#include <unordered_set>
using namespace std;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> hashSet;
        for (int elem:nums){
            if (hashSet.count(elem)==0){
                hashSet.insert(elem);
            }
            else{
                return true;
            }
        }

        return false;
    }
};